src/main/java/Duke.java

```diff
@@ -1,10 +1,264 @@
+                    }
+                } else if (input.split(" ")[0].equals("todo")) { // if task type is to-do
+                    try {
+                        if (input.contains(" ")) {
+                            String[] arr = input.split(" ", 2); // split input to get task item
+
+                            // add item to list
+                            klaun.addToList(new Todo(arr[1], "T")); // add to-do task to ls
+                        } else {
+                            throw new DukeException("invalid input: " + input);
+                        }
+                    } catch (Exception e) {
+                        System.out.println("??????????????????????????????????????????????????????????????\n");
+                        System.out.println("Oh no !! I think you forgot to add your todo description :O\n");
+                        System.out.println("??????????????????????????????????????????????????????????????\n");
+                    }
+                } else if (input.split(" ")[0].equals("deadline")) { // if task type is deadline
+                    try {
+                        if (input.contains(" ") && input.contains("/by")) {
+                            String[] arr = input.split("/by", 2); // split to get deadline of task
+                            String item = arr[0].split(" ", 2)[1]; // get item and remove "deadline" from string
+
+                            // add item to list
+                            klaun.addToList(new Deadline(item, "D", arr[1]));
+                        } else { // if format of deadline task is wrong
+                            throw new DukeException("invalid input: " + input);
+                        }
+                    } catch (Exception e) {
+                        System.out.println("??????????????????????????????????????????????????????????????\n");
+                        System.out.println("Oh no !! Your format should be \"deadline ____ /by ____\" \n");
+                        System.out.println("??????????????????????????????????????????????????????????????\n");
+                    }
+                } else if (input.split(" ")[0].equals("event")) { // if task type is deadline
+                    try {
+                        if (input.contains(" ") && input.contains("/at")) {
+                            String[] arr = input.split("/at", 2); // split to get time of task
+                            String item = arr[0].split(" ", 2)[1]; // get item and remove "deadline" from string
+
+                            // add item to list
+                            klaun.addToList(new Event(item, "E", arr[1]));
+                        } else { // if format of event task is wrong
+                            throw new DukeException("invalid input: " + input);
+                        }
+                    } catch (Exception e) {
+                        System.out.println("??????????????????????????????????????????????????????????????\n");
```

Comment:
could perhaps create a new Exception class to group this exception with the deadline equivalent